src/main/kotlin/uk/gov/justice/digital/hmpps/courtregister/service/CourtService.kt

```diff
@@ -13,7 +15,32 @@ class CourtService(private val courtRepository: CourtRepository) {
     return CourtDto(court)
   }
 
-  fun findAll(): List<CourtDto> {
+  fun findAll(activeOnly: Boolean = false): List<CourtDto> {
+    if (activeOnly) {
+      return courtRepository.findByActiveOrderById(true).map { CourtDto(it) }
+    }
     return courtRepository.findAll().map { CourtDto(it) }
   }
+
+  fun updateCourt(courtId: String, courtUpdateRecord: UpdateCourtDto): CourtDto {
```

Comment:
Do the updates need @Transactional?  Couldn't see an integration test for them so not sure...

Reply:
Working on integration tests now